### PR TITLE
test on arch

### PR DESCRIPTION
**Pull request by joshrabinowitz**
Test on Arch -- WIP

### CODE REVIEW

File: .ci/docker-ci/arch/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM archlinux:base-20220220.0.48372
+
+LABEL maintainer="mail@sobolevn.me"
+LABEL vendor="git-secret team"
+
+RUN pacman -Syu --needed --noconfirm \
```
Comment (sobolevn) [inline .ci/docker-ci/arch/Dockerfile:6]: What do we need to do to package `git-secret` for Arch? (I've last used it like 15 years ago)

**Review by sobolevn — COMMENTED** [review]
Thanks, cool!

File: .ci/docker-ci/arch/Dockerfile (hunk 1 above)
Comment (joshrabinowitz) [reply .ci/docker-ci/arch/Dockerfile:6]: I don't know! I guess we'll have to look at some examples and/or read up on Arch packaging a little. Perhaps someone can assist

Comment (joshrabinowitz) [reply .ci/docker-ci/arch/Dockerfile:6]: https://wiki.archlinux.org/title/creating_packages

**Pull request merged**